CHANGELOG.md

```diff
@@ -114,6 +114,7 @@ ___
 - Allow Cloud Validator `options` to be async (dblythy) [#7155](https://github.com/parse-community/parse-server/pull/7155)
 - Optimize queries on classes with pointer permissions (Pedro Diaz) [#7061](https://github.com/parse-community/parse-server/pull/7061)
 - Test Parse Server continuously against all relevant Postgres versions (minor versions), added Postgres compatibility table to Parse Server docs (Corey Baker) [#7176](https://github.com/parse-community/parse-server/pull/7176)
+- Catch and return validation error instead of crash [#7192](https://github.com/parse-community/parse-server/pull/7192)
```

Comment:
This needs some rewording.

Reply:
Could you be more specific on what do you think is missing?

Reply:
It just seems too unspecific, it does not mention afterSave or JSON validation or anything specific to this PR.

Reply:
This is fixed accordingly. With mention beforeSave and JSON validation